src/net.h

```diff
@@ -1048,17 +1046,7 @@ class CNode
      * in CConnman::AttemptToEvictConnection. */
     std::atomic<int64_t> nLastTXTime{0};
 
-    // Ping time measurement:
-    // The pong reply we're expecting, or 0 if no pong expected.
-    std::atomic<uint64_t> nPingNonceSent{0};
-    /** When the last ping was sent, or 0 if no ping was ever sent */
-    std::atomic<std::chrono::microseconds> m_ping_start{0us};
-    // Last measured round-trip time.
-    std::atomic<int64_t> nPingUsecTime{0};
-    // Best measured round-trip time.
     std::atomic<int64_t> nMinPingUsecTime{std::numeric_limits<int64_t>::max()};
```

Comment:
Genuine question, why should best ping time be in `CNode` but all the others in `Peer`? We use best ping time for eviction logic which seems like an application-layer thing...

Reply:
Answered here: https://github.com/bitcoin/bitcoin/pull/20721#issuecomment-752052378